packages/cli/src/CommandsLogic.ts

```diff
@@ -31,6 +31,10 @@ import { GSNContractsDeployment } from '@opengsn/common/dist/GSNContractsDeploym
 require('source-map-support').install({ errorFormatterForce: true })
 
 export interface RegisterOptions {
+  /** ms to sleep if waiting for RelayServer to set its owner */
+  sleep: number
```

Comment:
1. time should have `Ms` or `Sec` suffix
2. params should be more clear to their target
3. no values for real network.

So its better to have **setOwnerTimoutSec** (1 sec for test, 60sec for real network), and **setOwnerPollSec** (0.1sec on test, 10sec on real network)
The count is not interesting

Reply:
added parameters for 'gsn-relayer-register' (no automated tests, will test manually on register branch)